### DISCUSSION TITLE
Z-Homing doesn't work with BL Touch installed - Z is going up only

### DISCUSSION THREAD

**Discussion by joergensen70** [mriscoc/Ender3V2S1#944] [Q&A]
Hi all, hope somebody can help me here.
I have an Ender 3Pro with TJC Display and 4.27 Mainboard. Also a BL Touch is installed.

Everything was working fine with Firmware Ender3V2-427-BLTUBL-MPC-20230312 except some known issues about displaying bed level values
Now I was really looking forward for the new Release and installed Ender3V2-427-BLTUBL-MPC-20230522B and Z-Homing stopped working.  If I ever try to home Z it's only going up, next time up again and so on.
Exactly the same issue I had with the stock firmware from Ender after installing the BL Touch. I solved it with the firmware above but not the problem persist.
I did reset all settings and  stored them after update.

Anything I'm doing wrong here? Help would be appreciated

**Comment by mriscoc** [comment]
Re-check BLTouch wiring.

**Comment by joergensen70** [comment]
I did... Loaded the previous Firmware and everything works again.Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Miguel Risco-Castillo ***@***.***> Datum: 01.06.23  17:24  (GMT+01:00) An: mriscoc/Ender3V2S1 ***@***.***> Cc: joergensen70 ***@***.***>, Author ***@***.***> Betreff: Re: [mriscoc/Ender3V2S1] Z-Homing doesn't work with BL Touch
  installed - Z is going up only (Discussion #944) 
Re-check BLTouch wiring.

**Reply by mriscoc** [reply]
Please post here a video of your issue. Try disabling HS mode.

**Reply by joergensen70** [reply]
Tried to capture a Video. Therefore I flashed the Firmware once again. Now it's working for some reason. Didn't change anything. Very strange...

But with this SW I expected to see the numbers while leveling. This is still not the case for me.
But anyhow... Great SW, generations away from the stock SW. Awesome 👍

**Comment by mriscoc** [comment]
Currently there isn't a precompiled version for 4.2.7 board and TJC display.
https://github.com/mriscoc/Special_Configurations/releases/tag/tjc

You need to do a custom compile or request a precompiled firmware:
https://github.com/mriscoc/Special_Configurations/blob/main/README.md#how-to-request-a-custom-build

**Reply by joergensen70** [reply]
How can I request a precompiled Version? I'm quite new on this topic and hesitate to do my own build.

**Reply by joergensen70** [reply]
I did a donation through joe.....@gmx.de. I would like to request a custom build for my Ender 3 pro, pimpt with the TJC Didplay, a BT Touch and the 4.2.7 Mainboard. Would really love to see the numbers for the bed leveling. Would appreciate your help.
Thanks

**Reply by mriscoc** [reply]
> I did a donation through [joe.....@gmx.de](mailto:joe.....@gmx.de). I would like to request a custom build for my Ender 3 pro

Thank you! your firmware request will be ready soon.

**Comment by joergensen70** [comment]
Thank you in advance
JörgVon meinem/meiner 
Galaxy gesendet
